src/plugins/CaptionsPlugin.js

```diff
@@ -53,16 +79,150 @@ export class CaptionsPlugin extends ButtonPlugin {
       });
     }
 
+    this.fontSizeSelectors.forEach((selector) => {
```

Comment:
Could we pull this anonymous function into a class method? Might help readability IMO.